### PR TITLE
Prepare readme for 1.1.0, mark unofficial packages as unofficial

### PR DESCRIPTION
**Pull request by d-513**

### CODE REVIEW

**Comment by Scrumplex** [comment]
I don't like saying `official` and `unofficial`. Maybe we can use `upstream` and `community maintained`

**Comment by d-513** [comment]
> I don't like saying `official` and `unofficial`. Maybe we can use `upstream` and `community maintained`

what about now?

File: README.md [hunk 1]
```diff
@@ -109,28 +94,37 @@ If you are too lazy to do all these steps, you can just download [a prebuild x86
...
 - [Gzipped binaries](http://glowiak.github.io/file/polymc-latest-fbsd64-raw) - traditional way to distribute, unpack and run.
 
-In both cases you need X11, Qt5 and Java installed. Both files are 64bit only.
+In both cases you need X11, Qt5 and Java installed. Both files are 64bit only.  
+You can build from source - see [BUILD.md](./BUILD.md)
 
 ## <img src="https://raw.githubusercontent.com/AliasIO/wappalyzer/master/src/drivers/webextension/images/icons/OpenBSD%20httpd.svg" height="20" /> OpenBSD
 
-For OpenBSD available are [gzipped 32-bit binaries](http://glowiak.github.io/file/polymc-latest-obsd32-raw), download, unpack and run.
+There are community-maintained packages available:
```
Comment (Scrumplex) [inline README.md]:
```suggestion
There are community-maintained binary packages available:
```

**Review by Scrumplex — CHANGES_REQUESTED** [review]
apart from this, this looks fine!

**Comment by LennyMcLennington** [comment]
I don't see a problem with calling them official and unofficial or official and community maintained. "Upstream" makes less sense than "official".

**Comment by Scrumplex** [comment]
> I don't see a problem with calling them official and unofficial or official and community maintained. "Upstream" makes less sense than "official".

Official and unofficial is kinda heated in the context of a Qt-based Minecraft launcher. I mean just because we didn't make the packages ourselves doesn't mean they are less-official. If I were to provide binaries for Arch Linux on personal infrastructure I would just classify it as “community maintained”.

**Review by Scrumplex — APPROVED** [review]

**Review by timoreo22 — APPROVED** [review]

**Comment by LennyMcLennington** [comment]
> I mean just because we didn't make the packages ourselves doesn't mean they are less-official.

I disagree. We have control over the quality of official packages and updating them when they need updating. We don't have that control for community maintained packages. We should separate them as officially supported packages and community supported packages, and include contact methods for the maintainers of any community packages and explain that we may not be able to provide support for packaging issues with community packages.

**Comment by LennyMcLennington** [comment]
e.g. @glowiak who contributed the slackware and BSD packaging can be contacted by emailing [glowiak1111@yandex.com](mailto:glowiak1111@yandex.com), that should probably be mentioned.

File: README.md [hunk 2]
```diff
@@ -10,8 +10,6 @@ This is a **fork** of the MultiMC Launcher and not endorsed by MultiMC. The Poly
 <br>
 
 # Installation
-- All packages (archived by version) can be found [here](https://packages.polymc.org/) ([latest](https://packages.polymc.org/latest)).
```
Comment (ghost) [inline README.md:13]: why

**Comment by ghost** [comment]
also i should probably get BSD builds set up on Jenkins and we can have that available as well

File: README.md (hunk 2 above)
Comment (d-513) [reply README.md:13]: Because we are moving to GitHub releases, but I agree it should probably stay until that’s done

**Comment by Scrumplex** [comment]
> I disagree. We have control over the quality of official packages and updating them when they need updating. We don't have that control for community maintained packages.

Yes. I just dislike "unofficial", because everyone is free to package PolyMC however they want. Maybe we can just extend the issue templates to always include what distribution method was used (Maybe only list our "official" options and one called "Other (community maintained)")

File: README.md (hunk 2 above)
Comment (ghost) [reply README.md:13]: keep in mind jenkins contains generic linux builds that GHA doesn't have, and I'm going to implement BSD builds as well

Comment (Scrumplex) [reply README.md:13]: We build on Ubuntu 20.04 and produce a portable zip. Might also add system builds

**Review by txtsd — APPROVED** [review]

**Comment by LennyMcLennington** [comment]
> > I disagree. We have control over the quality of official packages and updating them when they need updating. We don't have that control for community maintained packages.
> 
> Yes. I just dislike "unofficial"

Well you can use any wording you want the difference still needs to be highlighted because we can't support every community maintained package like we can support the packages that we directly control... There are officially supported packages, and community supported packages. Just because you dislike it doesn't make it not true. I never said people can't package PolyMC, I said we can't officially support every package someone makes.....

**Comment by d-513** [comment]
@binex-dsk

**Comment by ghost** [comment]
ok

@glowiak I'm unfamiliar with BSD packaging. What binary formats are there that can be installed to the BSD system via a package manager, and how would I build one?

**Pull request merged**